pages/Schedules.tsx

```diff
@@ -0,0 +1,156 @@
+import { Page } from '@/components/Page/Page'
+import React from 'react'
+
+export default function Schedules() {
+  return (
+    <Page>
+      <section className="flex flex-col text-center w-full items-center">
+        <h1>Schedule</h1>
+        <h3>Schedule is in PST (Pacific Standard Time)</h3>
+        <h2>Saturday, April 2</h2>
+        <div className="flex flex-row gap-12 px-12 my-6">
+          <div className="flex flex-col w-full gap-5">
+            <div className="flex flex-col font-medium">Events</div>
+            <div className="flex items-center gap-5">
+              <div className="flex flex-col w-full text-left font-medium">
```

Comment:
Refactor into a Map and have a separate React.component that renders each column
example:

Code Pointers: 

- [keys of a map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys)
- [refactoring using a map](https://github.com/andreidimaano/portfoliov2/blob/main/scripts/casestudy.js)
- [rendering using a map](https://contactmentor.com/render-array-map-react-js/)

```
In a utils file:
const Schedule = new Map ([
    [ 
        'Activities',
        [
            {
                "event_name_here": {time: "7:30AM"}
            },
            {
                "event_name_here": {time: "7:30AM"}
            },
        ]
    ],
    [
         'Events',
        [
            {
                "event_name_here": {time: "7:30AM"}
            },
            {
                "event_name_here": {time: "7:30AM"}
            },
        ]
    ],
]);

```

```
in the component file:
return(
    <>
        {Object.keys(Schedule).map(() => {
            return(
                <>
                    <div>
                       your rectangles go here :)
                    </div>
                </>
            )
        })}
    </>
)
```